index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="coalabearDocsApp">
+
+<head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
+    <title>coala-bears Documentation</title>
+
+    <script src="https://use.fontawesome.com/663882be62.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext'>
+    <link rel='stylesheet' type='text/css' href='bower_components/materialize/dist/css/materialize.min.css'>
+    <link href="app/styles/main.css" rel="stylesheet">
+    <script src="bower_components/jquery/dist/jquery.min.js"></script>
```

Comment:
I think this change is temporary? You might wanna switch to cdns at the end?
Just confirming.